dot/network/notifications.go

```diff
@@ -241,7 +246,7 @@ func (s *Service) broadcastExcluding(info *notificationsProtocol, excluding peer
 			})
 
 			logger.Trace("sending handshake", "protocol", info.protocolID, "peer", peer, "message", hs)
-			err = s.host.send(peer, info.protocolID, hs)
+			err = s.host.send(peer, info.protocolID, hs, *stream)
```

Comment:
remove the stream here, the `peer` and `stream` do not correspond to the same peer, since we are iterating over all our peers